app/views/splash/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="splash-container">
+<div class="splash-title">
+<h1>Budget</h1>
+</div>
+    <div class="splash-inner">
+    <%= button_to 'LOG IN', new_user_session_path, method: :get, class: 'signin' %>
+    <%= link_to 'SIGN UP', new_user_registration_path, class: 'signup' %>
+    </div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Colors.-,Typography%3A%20font%20face%2C%20size%20and%20weight.,-Layout%3A%20composition%20and), Please add the correct font families for all elements of all pages in your app which is `proxima nova` light and bold.

![image](https://github.com/MayPyone/Budget/assets/68796232/a1ed2971-6614-4c14-8121-53db14f8b62a)